src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -248,19 +275,68 @@ public Config(IGerritHudsonTriggerConfig config) {
         enableProjectAutoCompletion = config.isEnableProjectAutoCompletion();
         projectListFetchDelay = config.getProjectListFetchDelay();
         projectListRefreshInterval = config.getProjectListRefreshInterval();
-        if (config.getCategories() != null) {
-            categories = new LinkedList<VerdictCategory>();
-            for (VerdictCategory cat : config.getCategories()) {
-                categories.add(new VerdictCategory(cat.getVerdictValue(), cat.getVerdictDescription()));
-            }
-        }
+
+        assertDefaultCategories();
+
         if (config.getReplicationConfig() != null) {
             replicationConfig = new ReplicationConfig(config.getReplicationConfig());
         }
         watchdogTimeoutMinutes = config.getWatchdogTimeoutMinutes();
         watchTimeExceptionData = addWatchTimeExceptionData(config.getExceptionData());
     }
 
+    public static Integer getIntegerFromString(String str) {
+        if (str != null && !str.equals(""))
+            return Integer.parseInt(str);
+        return null;
+    }
+
+    // make sure that there is a CR and VER category
```

Comment:
[minor] let's called the `Code-Review` and `Verified`

Reply:
Done